PlaceRouteHierFlow/router/GcellGlobalRouter.cpp

```diff
@@ -1429,14 +1429,21 @@ int  GcellGlobalRouter::JudgeSymmetry(std::vector<std::pair<int,int> > &path,std
 
 };
 
+void routerlpsolvelogger(lprec *lp, void *userhandle, char *buf)
+{
+  // Strip leading newline
+  while((unsigned char)*buf == '\n') buf++;
+  // Log non-empty lines
+  if (*buf != '\0') std::cout << "GcellGlobalRouter lpsolve: " << buf << std::endl;
```

Comment:
@854768750 GcellGlobalRouter still has some 'cout <<' and 'cerr <<' lines. Could you please take a look?

Reply:
The latest commit has changed these lines.